packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -30,31 +32,54 @@ import ReactDOM from "react-dom";
 import React from "react";
 import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
 import {
+    createColorPalette,
     DesignSystem,
     DesignSystemDefaults,
+    designUnit,
+    height,
+    horizontalSpacing,
 } from "@microsoft/fast-components-styles-msft";
-import { TabsItem } from "@microsoft/fast-components-react-base";
+import {
+    LabelClassNameContract,
+    ListboxItemProps,
+    TabsItem,
+} from "@microsoft/fast-components-react-base";
 import style from "./explorer.style";
 import {
     Background,
     DarkModeBackgrounds,
+    Label,
     LightModeBackgrounds,
     Pivot,
     PivotClassNameContract,
+    Select,
+    SelectOption,
+    Toggle,
+    ToggleClassNameContract,
 } from "@microsoft/fast-components-react-msft";
 import { ViewerManagedClasses } from "@microsoft/fast-tooling-react/dist/viewer/viewer/viewer.props";
 import {
     FormChildOptionItem,
     FormClassNameContract,
 } from "@microsoft/fast-tooling-react/dist/form/form";
 import { Prism as SyntaxHighlighter } from "react-syntax-highlighter";
-import jsx from "react-syntax-highlighter/dist/esm/languages/prism/jsx";
 import syntaxHighlighterStyles from "./syntax-highlighting-style";
 import { childOptions, history, menu } from "./config";
 import * as componentViewConfigs from "./utilities/configs";
 import { Scenario } from "./utilities/configs/data.props";
 import { MemoizedFunction } from "lodash";
-import { Direction } from "@microsoft/fast-web-utilities";
+import { Direction, format } from "@microsoft/fast-web-utilities";
+import {
+    ColorHSL,
+    ColorRGBA64,
+    hslToRGB,
+    parseColor,
+    rgbToHSL,
+} from "@microsoft/fast-colors";
+import { toPx } from "@microsoft/fast-jss-utilities";
+
+const dark: string = "#111111";
+const light: string = "#FFFFFF";
```

Comment:
should we be taking these from the design system?

Reply:
How? you need to input a background value into the DesignSystem in order for it to generate the color values, We could use DesignSystemDefaults.backgroundColor for `const light` but we would still need to pass a hard coded value to `const dark`

Reply:
We do have a const `black` and a `white` in msft styles in the `color.constants.ts` file but they are not exported by the package.

Reply:
Maybe we should be, but if we aren't now then no problem.